### PR TITLE
[#176] Use bitnami chart repo URL in Ditto's requirements

### PR DESCRIPTION
**Pull request by sophokles73**

### CODE REVIEW

File: charts/ditto/requirements.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@
 ---
 dependencies:
   - name: mongodb
-    repository: https://kubernetes-charts.storage.googleapis.com
+    repository: https://charts.bitnami.com/bitnami
     version: 7.6.3
```
Comment (thjaeckle) [inline charts/ditto/requirements.yaml:15]:
```suggestion
    version: ~7.14.7
```
Should we use the same version as Hono defines?

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Thank you for fixing this, @sophokles73 
The deprecation seems indeed to be the issue here.

I think it makes sense to define the same MongoDB version as Hono defines in order to be sure that a working MongoDB version is referenced ;)

File: charts/ditto/requirements.yaml (hunk 1 above)
Comment (sophokles73) [reply charts/ditto/requirements.yaml:15]: Well, I am glad to align the versions. However, you will need to tell me if Ditto will work with it ...

Comment (thjaeckle) [reply charts/ditto/requirements.yaml:15]: Yes, it will - this references the latest MongoDB 4.2 which Ditto 1.5.0 was explicitly tested against.

Comment (sophokles73) [reply charts/ditto/requirements.yaml]: done

**Review by thjaeckle — APPROVED** [review]

**Pull request merged**